src/main/java/frc/robot/subsystems/Storage.java

```diff
@@ -3,14 +3,22 @@
 import com.revrobotics.CANSparkMax;
 import com.revrobotics.CANSparkBase.IdleMode;
 import com.revrobotics.CANSparkLowLevel.MotorType;
+import edu.wpi.first.wpilibj.DigitalInput;
 
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
 import frc.robot.constants.StorageConstants;
 
 public class Storage extends SubsystemBase {
 
-    private final CANSparkMax storageMotor = new CANSparkMax(StorageConstants.motorPort,
+    private final CANSparkMax transitionMotor = new CANSparkMax(StorageConstants.transitionMotorPort,
             MotorType.kBrushless);
+    private final CANSparkMax leftStorage = new CANSparkMax(
```

Comment:
Add "motor" to the variable names for "leftStorage" and "rightStorage."

Reply:
I agree